packages/engine/src/scene/functions/SceneLoading.ts

```diff
@@ -275,6 +276,10 @@ export class WorldScene {
         createParticleEmitterObject(entity, component.data)
         break
 
+      case 'cloud':
+        addObject3DComponent(entity, { obj3d: Clouds, objArgs: component.data })
```

Comment:
this is outdated, please merge from dev. we no longer apply parameters in this way, it is all explicit. this way, we ensure typings are correct and there is no ambiguity about where values come from.